bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/config/rm-config.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+	https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:broadlink:rmconfig">
+		<parameter name="ipAddress" type="text">
+			<context>network-address</context>
+			<label>Network Address</label>
+			<description>Network address of the device.</description>
+			<required>true</required>
+		</parameter>
+		<parameter name="staticIp" type="boolean">
+			<context>static-ip</context>
+			<label>Static IP</label>
+			<description>Will the device always be given this network address?</description>
+			<required>true</required>
+			<advanced>true</advanced>
+			<default>true</default>
+		</parameter>
+		<parameter name="port" type="integer">
+			<default>80</default>
+			<context>network-port</context>
+			<label>Network Port</label>
+			<description>Network port of the device.</description>
+			<required>true</required>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="macAddress" type="text">
+			<context>mac-address</context>
+			<label>MAC Address</label>
+			<description>MAC address of the device.</description>
+			<required>true</required>
```

Comment:
A lot of people do not know what a mac address is, or how to find it. Is this needed or can be it discovered by the binding? You have the mac address as a property and since it can not change I have a feeling (not 100% sure) that it should be a property and not a config. Understand I am not an expert so if you think I am wrong feel free to leave it until the next person comes to review.

Reply:
Yeah unsure about this. `macAddress` has been a config field since I "inherited" the binding. Looking at other bindings, they seem to declare it in XML like this too. Happy to defer to a framework expert on the best course of action here